JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using JDI.Light.Elements.Base;
+using JDI.Light.Elements.Common;
+using JDI.Light.Exceptions;
+using JDI.Light.Factories;
+using JDI.Light.Interfaces.Complex;
+using OpenQA.Selenium;
+
+namespace JDI.Light.Elements.Complex
+{
+    public class RadioButtons : UIElement, IRadioButtons
+    {
+        public By RadioButtonLocator { get; set; }
+
+        public By LabelLocator { get; set; }
+
+        private List<Label> Labels => FindElements(LabelLocator)
+            .Select(element => UIElementFactory.CreateInstance<Label>(LabelLocator, this, element)).ToList();
+
+        private List<UIElement> Radios => FindElements(RadioButtonLocator)
+            .Select(element => UIElementFactory.CreateInstance<UIElement>(RadioButtonLocator, this, element)).ToList();
+
+        protected RadioButtons(By byLocator) : base(byLocator)
+        {
+            RadioButtonLocator = By.CssSelector("[type='radio']");
+            LabelLocator = By.CssSelector(".html-left > label");
+        }
+
+        public By RadioLocator { get; set; }
+
+        public void Select(string name)
+        {
+            var element = Labels.FirstOrDefault(label => label.Text.Trim() == name);
+            if (element == null)
+            {
+                throw new ElementNotFoundException($"label: {name} not found");
+            }
+
+            element.Click();
+        }
+
+        public void Select(int index)
+        {
+            Labels[index - 1].Click();
+        }
+
+        public new string Selected()
+        {
+            var checkedId = Radios.First(IsChecked).GetAttribute("id");
+
+            return Labels.First(label => checkedId.Contains(label.GetAttribute("for")))?.Text;
+        }
+
+        public List<string> Values()
+        {
+            return Labels.Select(label => label.Text.Trim()).ToList();
+        }
+
+        public string Value => Selected();
+
+        private static bool IsChecked(UIElement radioButton) => radioButton.GetAttribute("checked") != null;
```

Comment:
Please do not use a static methods.
Also we don't need this method at all, looks like it is used only once.

Reply:
method removed